### PR TITLE
main-navigation-icons-cy-95

### PR DESCRIPTION
**Pull request by cris22380**

### CODE REVIEW

File: src/icons/IconCross.tsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import styled from "styled-components";
+const IconBase = styled.svg<{size: number}>`
```
Comment (diegomart2000) [inline src/icons/IconCross.tsx]: This doesn't seem ok here.

File: src/icons/IconDocumentsComponents.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <path fillRule="evenodd" clipRule="evenodd" d="M22.8771 20.8825C22.3461 20.3515 22.3461 19.4906 22.8771 18.9597L23.8385 17.9982C24.3695 17.4673 25.2303 17.4673 25.7613 17.9982L26.7227 18.9597C27.2537 19.4906 27.2537 20.3515 26.7227 20.8825L25.7613 21.8439C25.2303 22.3749 24.3695 22.3749 23.8385 21.8439L22.8771 20.8825ZM22.8771 29.0404C22.3461 28.5094 22.3461 27.6485 22.8771 27.1175L23.8385 26.1561C24.3695 25.6251 25.2303 25.6251 25.7613 26.1561L26.7227 27.1175C27.2537 27.6485 27.2537 28.5094 26.7227 29.0404L25.7613 30.0018C25.2303 30.5328 24.3695 30.5328 23.8385 30.0018L22.8771 29.0404ZM26.956 24.9614C26.425 24.4305 26.425 23.5696 26.956 23.0386L27.9174 22.0772C28.4484 21.5462 29.3093 21.5462 29.8403 22.0772L30.8017 23.0386C31.3326 23.5696 31.3326 24.4305 30.8017 24.9614L29.8403 25.9228C29.3093 26.4538 28.4484 26.4538 27.9174 25.9228L26.956 24.9614ZM18.7981 24.9614C18.2672 24.4305 18.2672 23.5696 18.7981 23.0386L19.7595 22.0772C20.2905 21.5462 21.1514 21.5462 21.6824 22.0772L22.6438 23.0386C23.1748 23.5696 23.1748 24.4305 22.6438 24.9614L21.6824 25.9228C21.1514 26.4538 20.2905 26.4538 19.7595 25.9228L18.7981 24.9614Z" fill="#172C55"/>
+      </SIconBase>,
+}
+
```
Comment (diegomart2000) [inline src/icons/IconDocumentsComponents.tsx:30]: Please remove unnecessary spaces.

File: src/icons/IconTerminal.tsx [hunk 3]
```diff
@@ -1,22 +1,86 @@
...
+import { SIconBase } from "./styles/SIconBase";
+import { IIconSize } from "../utils/types";
 
-export function IconTerminal() {
-  return (
-    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+const IconSize = {
+  16: (
+    <SIconBase width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M8.66663 10H10.6666" stroke="#172C55" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round" />
```
Comment (diegomart2000) [inline src/icons/IconTerminal.tsx]: You should remove the stroke color inline style which is coming from base icon.

Comment (diegomart2000) [reply src/icons/IconTerminal.tsx]: This is a comment that applies to all the icons.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by cris22380** [comment]
Screen Shot 2021-12-06 at 14 42 58
Screen Shot 2021-12-06 at 14 43 15
Screen Shot 2021-12-06 at 14 43 29
Screen Shot 2021-12-06 at 14 43 39
Screen Shot 2021-12-06 at 14 43 52
Screen Shot 2021-12-06 at 14 25 17
Screen Shot 2021-12-06 at 14 27 04
Screen Shot 2021-12-06 at 14 27 33
Screen Shot 2021-12-06 at 14 28 04
Screen Shot 2021-12-06 at 14 31 02
Screen Shot 2021-12-06 at 14 32 41
Screen Shot 2021-12-06 at 14 32 54
Screen Shot 2021-12-06 at 14 33 06
Screen Shot 2021-12-06 at 14 33 17
Screen Shot 2021-12-06 at 14 33 31
Screen Shot 2021-12-06 at 14 34 29
Screen Shot 2021-12-06 at 14 34 39
Screen Shot 2021-12-06 at 14 34 53
Screen Shot 2021-12-06 at 14 35 02
Screen Shot 2021-12-06 at 14 35 24
Screen Shot 2021-12-06 at 14 36 27
Screen Shot 2021-12-06 at 14 36 37
Screen Shot 2021-12-06 at 14 36 46
Screen Shot 2021-12-06 at 14 36 56
Screen Shot 2021-12-06 at 14 37 08
Screen Shot 2021-12-06 at 14 38 14
Screen Shot 2021-12-06 at 14 38 28
Screen Shot 2021-12-06 at 14 38 40
Screen Shot 2021-12-06 at 14 38 52
Screen Shot 2021-12-06 at 14 39 07
Screen Shot 2021-12-06 at 14 41 06
Screen Shot 2021-12-06 at 14 41 18
Screen Shot 2021-12-06 at 14 41 31
Screen Shot 2021-12-06 at 14 41 44
Screen Shot 2021-12-06 at 14 41 57
Screen Shot 2021-12-06 at 14 44 57
Screen Shot 2021-12-06 at 14 45 13
Screen Shot 2021-12-06 at 14 45 40
Screen Shot 2021-12-06 at 14 45 56
Screen Shot 2021-12-06 at 14 46 24
Screen Shot 2021-12-06 at 14 49 07
Screen Shot 2021-12-06 at 14 49 37
Screen Shot 2021-12-06 at 14 49 52
Screen Shot 2021-12-06 at 14 50 38
Screen Shot 2021-12-06 at 14 50 52
Screen Shot 2021-12-06 at 14 52 28
Screen Shot 2021-12-06 at 14 52 50
Screen Shot 2021-12-06 at 14 53 10
Screen Shot 2021-12-06 at 14 53 57
Screen Shot 2021-12-06 at 14 54 08
Screen Shot 2021-12-06 at 14 56 29
Screen Shot 2021-12-06 at 14 56 38
Screen Shot 2021-12-06 at 14 56 59
Screen Shot 2021-12-06 at 14 57 31
Screen Shot 2021-12-06 at 14 57 46

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**